app/views/tag/index.html.erb

```diff
@@ -58,5 +58,12 @@
 
 <div class="col-md-6">
 <br><br>
-<iframe src="stats/graph.html?limit=50" height="500" width="100%" frameborder="0" ></iframe>
+<iframe id="myImage" src="stats/graph.html?limit=50" height="500" width="100%" frameborder="0" ></iframe>
+<h6 class="text-right" style="color:#ccc;">Tags most commonly used, linked by how often they are used together.<br>
+  View <a style="cursor: pointer;" onclick="document.getElementById('myImage').src='stats/graph.html?limit=50'"><u>50</u></a> |
+  <a style="cursor: pointer;" onclick="document.getElementById('myImage').src='stats/graph.html?limit=100'"><u>100</u></a> |
+  <a style="cursor: pointer;" onclick="document.getElementById('myImage').src='stats/graph.html?limit=250'"><u>250</u></a> |
+  <a style="cursor: pointer;" onclick="document.getElementById('myImage').src='stats/graph.html?limit=500'"><u>500</u></a>
+</h6>
```

Comment:
Please indent the code.

Reply:
wow this is such a clever solution! Thank you!

Reply:
Thank you @jywarren @gauravano!